Zotero RDF.js

```diff
@@ -244,7 +244,7 @@ function generateItem(item, zoteroType, resource) {
 		// set series type
 		Zotero.RDF.addStatement(series, rdf+"type", n.bib+"Series", false);
 		// add relationship to resource
-		Zotero.RDF.addStatement((containerElement ? containerElement : resource), n.dcterms+"isPartOf", series, false);
+		Zotero.RDF.addStatement(resource, n.dcterms+"isPartOf", series, false);
```

Comment:
Thanks but I don't think this is the right fix -- if there is a container (i.e. a book or a journal) the series term should be applied to that with the isPartOf term, not to the article/chapter. This needs to get fixed on import, not on export.